documents/operating.md

```diff
@@ -52,14 +52,223 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
 SECTION 5: 🗣🗣🗣 Governing Process 🗣🗣🗣
 ---------------------------------------------------------------------
-- [ ] Types of decision
-- [ ] Records and documentation
-- [ ] Membership
-- [ ] Mechanisms for deliberation
-  - [ ] ... On GitHub
-  - [ ] ... On Slack
-- [ ] Use of emoji
-- [ ] Meetings
+- [x] Types of decision
+- [x] Records and documentation
+- [x] Mechanisms for deliberation
+  - [x] ... On GitHub
+  - [x] ... On Slack
+  - [x] ... Plotting Sessions
+- [x] Use of emoji
+
+### Schedules and Expectations
+All meaningful organizational decisions will happen in monthly Plotting
+Sessions, which are to be scheduled and announced by the Overmind on
+April 1st of every year.  The schedule can be updated as needed, but
+a notice of 10 days must be given before any Plotting Session can be
+held.
+
+April 1st marks the beginning of the governing year, and officer elections
+are to be held during the month of April.
+
+
+### Deliberation Processes
+#### Plotting Session
+Plotting Sessions are agenda driven and require a Quorum.  They
+are led by the chair (as determined by the chain of command).
```

Comment:
"Chain of Command" could be a definition in Section 0, or it could just be laid out in this section.  I'm thinking basically: Overmind -> Undermind -> Secretary of Lore -> Treasure Goblin

And if Secretary of Lore is running a meeting, they delegate someone to take notes on their behalf.